scripts/update-ssh.sh

```diff
@@ -19,7 +19,7 @@ if test -f "$cache" -a -n "${TEST}"; then
     cat "$cache" >> "$tmp"
 else
     # add current gh keys into $tmp
-    ssh-import-id "gh:$user" --output "$tmp"
+    curl -s "https://github.com/$user.keys" -o "$tmp"
```

Comment:
if the user already has an explicit key from this repo, will this do the right thing (i.e. append to the file)?

Reply:
Thanks, fixed with `... >> $tmp`